src/main/java/com/amazon/opendistroforelasticsearch/security/filter/OpenDistroSecurityFilter.java

```diff
@@ -355,6 +356,9 @@ public void onFailure(Exception e) {
                 listener.onFailure(new ElasticsearchSecurityException(err, RestStatus.FORBIDDEN));
                 return;
             }
+        } catch (ElasticsearchException err) {
+            log.debug(err);
```

Comment:
Will this log exception? Please provide sample output.

Reply:
Changed it to log exception. The output is: `Unexpected exception [_] InvalidIndexNameException[Invalid index name [_], must not start with '_'.]`

Reply:
I followed the existing code style of [this line](https://github.com/opendistro-for-elasticsearch/security/blob/f3e474c1783e7afc592dcab4f6b72806abc47281/src/main/java/com/amazon/opendistroforelasticsearch/security/filter/OpenDistroSecurityFilter.java#L370). Does this look better now?